### ISSUE TITLE
Feature request: last N messages

### ISSUE THREAD

**Issue by gamesbook** [dubek/rabbitmq-dump-queue#4]
I am not sure this is possible, but - is it possible to extract the most recent N messages from a queue?  
So, if I had downloaded 20,000 & then 5,000 new ones arrived afterwards; then this would avoid having to download all of the previous ones again.

**Comment by dubek** [comment]
Thanks for the suggestion @gamesbook ! Unfortunately, I don't think RabbitMQ supports access to messages which are not at the head of the queue. Do you know about such option in the protocol / API? Through the RabbitMQ management GUI?

**Comment by gamesbook** [comment]
@dubek No, I do not know of such a protocol - I was being hopeful that there was such a status as "last in" for a message.  I will need to rethink my strategy on the project.

**Comment by dubek** [comment]
@gamesbook I'm not sure if it helps, but take a look at http://qdb.io/

**Comment by gamesbook** [comment]
Looks interesting, thanks.
